packages/wonder-blocks-link/custom-snapshot.test.js

```diff
@@ -46,7 +40,7 @@ describe("LinkCore", () => {
                                     light={light}
                                     {...stateProps}
                                     {...defaultHandlers}
-                                    tabIndex={stateProps.disabled ? -1 : 0}
```

Comment:
🤷‍♂️ Hmm - this changed?

Reply:
This is b/c our Link component doesn't actually support a disabled state. The reason for keeping tabIndex is a bit roundabout but it boils down to the ClickableHandlers type including tabIndex as a mandatory prop. Maybe ClickableHandlers isn't the right place for tabIndex. Since tabIndex is either 0 (disabled=false) or -1 (disabled=true) it can be derived from disabled and maybe each component that uses ClickableBehavior should be responsible for doing that mapping and assigning tabIndex to whatever components it renders.